app/src/main/java/com/lab/keyneez/presentation/main/like/LikeFragment.kt

```diff
@@ -7,8 +7,17 @@ import com.lab.keyneez.databinding.FragmentLikeBinding
 import com.lab.keyneez.util.binding.BindingFragment
 
 class LikeFragment : BindingFragment<FragmentLikeBinding>(R.layout.fragment_like) {
+    private var _binding: FragmentLikeBinding? = null
+        get() = requireNotNull(_binding) { "binding value is null." }
+
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        // 여기서 코드 작성
+    }
+
+    override fun onDestroyView() {
+        super.onDestroyView()
+        _binding = null
     }
```

Comment:
```suggestion
class LikeFragment : BindingFragment<FragmentLikeBinding>(R.layout.fragment_like) {
    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        // 여기서 코드 작성
    }
```

LikeFragment가 상속받는 BindingFragment를 보시면 binding 변수를 생성하고 `onDestroyView`에서 binding 변수를 null로 초기화하는 코드가 이미 포함되어 있어요! 중복되는 코드들은 지워주세요 :)

Reply:
넵 완료!